demos/drawer/permanent-drawer-above-toolbar.html

```diff
@@ -156,6 +150,9 @@ <h1 class="mdc-typography--display1">Permanent Drawer</h1>
           </div>
         </div>
 
+        <div class="extra-content-wrapper">
+          <button type="button" class="mdc-button mdc-button--stroked mdc-button--dense demo-toolbar-example-heading__rtl-toggle-button">Toggle RTL</button>
+        </div>
         <div class="extra-content-wrapper">
           <button id="toggle-wide">Toggle extra-wide content</button>
```

Comment:
`mdc-button` classes? (Ditto for "extra-tall content" button)

Reply:
ah weird...i did it in the other one. Changing